src/protocols/cosmos/scaffold/manifest.js

```diff
@@ -0,0 +1,16 @@
+const source = () => `
+      startBlock: 0`
```

Comment:
Isn't `startBlock` zero by default? 🤔

Reply:
So you're suggesting to remove `startBlock` entirely (as it's `0` by default)?

Reply:
Yes!